lib/components/viewers/trip-viewer.js

```diff
@@ -175,14 +181,10 @@ class TripViewer extends Component {
 
                 let stopLabel = null
                 if (viewedTrip.fromIndex === i) {
-                  stopLabel = intl.formatMessage({
-                    id: 'components.tripviewer.startOfTrip'
-                  })
+                  stopLabel = 'components.TripViewer.startOfTrip'
                 }
                 if (viewedTrip.toIndex === i) {
-                  stopLabel = intl.formatMessage({
-                    id: 'components.tripviewer.endOfTrip'
-                  })
+                  stopLabel = 'components.TripViewer.endOfTrip'
```

Comment:
You want to keep the id value together with `<FormattedMessage>` (or with intl.formatMessage), so that the formatjs CLI tool can pick up the string to translate. (In other words, don't use dynamic ids - the exception is for using ids defined in config.)

Reply:
Good to know! Thanks! Fixed in 67caf50f05cfa00f575984c8559b788eb79f36e8